src/hotspot/share/oops/klass.hpp

```diff
@@ -174,18 +174,20 @@ class Klass : public Metadata {
 
 #if INCLUDE_CDS
   // Various attributes for shared classes. Should be zero for a non-shared class.
-  u2 _shared_class_flags;
-  enum CDSSharedClassFlags {
+  u2 _aot_class_flags;
+  enum  {
     _in_aot_cache                          = 1 << 0,
     _archived_lambda_proxy_is_available    = 1 << 1,
     _has_value_based_class_annotation      = 1 << 2,
     _verified_at_dump_time                 = 1 << 3,
     _has_archived_enum_objs                = 1 << 4,
-    // This class was not loaded from a classfile in the module image
-    // or classpath.
-    _is_generated_shared_class             = 1 << 5,
-    // archived mirror already initialized by AOT-cache assembly: no further need to call <clinit>
-    _has_aot_initialized_mirror            = 1 << 6,
+    _is_generated_shared_class             = 1 << 5, // This class was not loaded from a classfile in the module image
```

Comment:
does it make sense to rename `_is_generated_shared_class` to `_is_generated_aot_class`?